android/titanium/src/java/org/appcelerator/titanium/io/TiFile.java

```diff
@@ -95,6 +83,35 @@ public boolean isWriteable()
 		return file.canWrite();
 	}
 
+    public boolean append(Object data) throws IOException {
```

Comment:
Is it necessary to have the first append(Object data) method in your example because I could overload append method but each one takes different parameter types?